pkg/virt-operator/kubevirt_test.go

```diff
@@ -1336,6 +1362,8 @@ var _ = Describe("KubeVirt Operator", func() {
 			addPodsAndPodDisruptionBudgets(defaultConfig)
 			makeApiAndControllerReady()
 			makeHandlerReady()
+			fakeNamespaceModificationEvent()
```

Comment:
I am always a little bit cautious with controllers. Can we have one unit tests where we check that we don't patch if the label is already present? That shows that we do definitely not hot-looping which can sometimes be hard to track.